todo_app/templates/index.html

```diff
@@ -10,9 +10,38 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <table class="table mb-4">
+        <tr>
+          <th>Title</th>
+          <th>Status</th>
+          <th></th>
+        </tr>
+        {% for item in items %}
+          <tr class="{{ 'warning' if item['status'] == 'Not Started' else 'info' }}">
+            <td>{{ item.title }}</td>
+            <td>
+              {% if item.status == "Not Started"%}
+              <form action="items/complete/{{item.id}}", method="POST">
+                <input type="submit" value="Mark as complete">
+              </form>
+              {% else %}
+              {{ item.status }}
+              {% endif %}
+            </td>
+            <td>
+              <form action="items/remove/{{item.id}}", method="POST">
+                <input type="submit" value="Remove" style="background-color: red;">
```

Comment:
entirely optional but since we've got bootstrap you could make a better-looking (albeit basic) button with `class="btn btn-danger"`